pull_request_template.md

```diff
@@ -1,34 +1,57 @@
 ### 📃 Summary
 <!-- Enter a short description about what this PR contains. -->
-
+---
 ### 🔍 Changeset
 <!-- List out the code changes introduced by this PR. Data Migrations, Endpoints, etc -->
-
+---
 ### 🏷 JIRA Task
 <!-- Provide a link to the associated JIRA task. -->
-
+---
 ### ↩️ Depends On
 <!-- Provide a link to any other PR dependencies. -->
-
+---
 ### 📸 Screenshots (before/after)
 <!-- If this PR alters any UI, provider before/after screenshots. -->
-
+---
 ### 🧪 QA
 <!-- List out the steps to test the feature. -->
 
+- Unit tests
+  - [ ] Yes
+  - [ ] No
+ 
+- Regression tests
+  - [ ] Yes
+  - [ ] No
+ 
+- Performance tests
+  - [ ] Yes
+  - [ ] No
+---
 ### ✨ Style Guide
-[API Style Guide on Confluence](https://wheelhealth.atlassian.net/wiki/spaces/TH/pages/2497871875/Care+API+Code+Style+Guide)
-
-[We're also using eslint/prettier to manage coding styles](https://github.com/heydoctor/eslint-config)
 
+- Prettier
+  - [ ] Yes
+  - [ ] No
+---
 ### 📈 Risk Analysis
 <!-- Provide a risk description -->
 
-- [ ] Does this affect patients?
-- [ ] Does this affect clinical?
-- [ ] Does this mutate data?
-- [ ] Does it touch PHI?
-- [ ] Does it touch payments?
-
+- Does this affect patients?
+  - [ ] Yes
+  - [ ] No
+- Does this affect clinical?
+  - [ ] Yes
+  - [ ] No
+- Does this mutate data?
+  - [ ] Yes
+  - [ ] No
+- Does it touch PHI?
+  - [ ] Yes
+  - [ ] No
```

Comment:
Can we elaborate on this? What exactly does "touching" PHI entail - a vast majority of Care PRs involve interacting at some level with a patient or a visit, trying to understand if this would just always be the case